src/connection.js

```diff
@@ -138,18 +138,6 @@ class Connection {
     //TODO
   }
 
-  escape(value) {
```

Comment:
Wouldn't it better to leave all these empty? or does that have the same effect?

Reply:
no, that haven't the same effect : 
if you call connection.format(...) this will throw an exception in log if not defined, like "TypeError: connection.format is not a function and stop script."
if defined :
`
var formatedQuery = connection.format('my query ??', [values]);
var another = formatedQuery + 'ff';
`
connection format will work, but the result will be undefined => error will be less clear to found for user. 

The only real possibility is to keep them and throw a message explaining the error.

Reply:
Well, I was thinking about returning string as is. But unlikely we can safely do that. So, right. And maybe throwing exception would be slightly better, but not that much.
But we are slightly breaking compatibility with existing connectors, right?

Reply:
but, right, better to have an error message for that.
changed : https://github.com/rusher/mariadb-connector-nodejs/commit/fd1d66afe9d50943848aa0a3c24820fad3f620ac